_data/team/roles/community_engagement.yml

```
@@ -1,7 +1,10 @@
 role: Community engagement
 tasks: 
-  - Responding to questions on Discord and GitHub Discussions
-  - Managing and triaging conversations on Discord and GitHub Discussions
+  - Responding to questions on [Discord]({{ site.discord.url }}) and [GitHub Discussions]({{ site.mailinglists.discussion.url }})
+  - Managing and triaging conversations on [Discord]({{ site.discord.url }}) and [GitHub Discussions]({{ site.mailinglists.discussion.url }})
+  - General management and administration of social accounts (i.e., [LinkedIn]({{ site.linkedin.url}}), [Bluesky]({{ site.bluesky.url }})), [website]({{ site.url }}), and [blog]({{ site.blog }}), including...
```

Removed the trailing "including...": This list seems sufficient.
```suggestion
  - General management and administration of social accounts (i.e., [LinkedIn]({{ site.linkedin.url}}), [Bluesky]({{ site.bluesky.url }})), [website]({{ site.url }}), and [blog]({{ site.blog }})
```